File: module7/Collaboration/Python/ListMessages.py
import requests

url = "https://api.ciscospark.com/v1/messages"

querystring = {"roomId":"<room ID>"}

headers = {
    'content-type': "application/json; charset=utf-8",
    'authorization': "Bearer <your token>",
    'cache-control': "no-cache",
    'postman-token': "a080dc60-da23-c605-c0ca-052d55ff5858"
    }

response = requests.request("GET", url, headers=headers, params=querystring)

print(response.text)

File: module7/Collaboration/NodeJS/ListMessages.js
var request = require("request");

var options = { method: 'GET',
  url: 'https://api.ciscospark.com/v1/messages',
  qs: { roomId: 'Y2lzY29zcGFyazovL3VzL1JPT00vYTliZTQyZTAtMzY3OS0xMWU2LTliNmItZTcxYjVhMmYyMzEy' },
  headers: 
   { 'postman-token': 'c850d810-4350-9012-ded1-6e23fd28f5cc',
     'cache-control': 'no-cache',
     authorization: 'Bearer NTIyN2U5NGQtY2JjZi00NGYxLWE0OTMtYzMwYTY3OWQzMDljOGE0YzA3ZTctNjY1',
     'content-type': 'application/json; charset=utf-8' } };

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});
